configs/config/pretrain/vision_transformer/supervised/supervised_16gpu_deit_b_example.yaml

```diff
@@ -16,11 +16,12 @@ config:
       LABEL_SOURCES: [disk_folder]
       DATASET_NAMES: [imagenet1k_folder]
       BATCHSIZE_PER_REPLICA: 32
+      DROP_LAST: True
       TRANSFORMS:
         - name: RandomResizedCrop
           size: 224
         - name: RandomHorizontalFlip
-        - name: VisslRandAugment
+        - name: RandAugment
```

Comment:
we should rebase to master. This change should be essentially reverted after the rebase.